src/modules/75-cf/pages/onboarding/OnboardingDetailPage.tsx

```diff
@@ -111,116 +112,145 @@ export const OnboardingDetailPage: React.FC = () => {
-          </Heading>
-        )}
-
-        {selectedStep === STEP.CREATE_A_FLAG && (
-          <CreateAFlagView selectedFlag={selectedFlag} setSelectedFlag={setSelectedFlag} />
-        )}
-
-        {selectedStep === STEP.SELECT_ENV_SDK && selectedFlag && (
-          <SetUpYourApplicationView
-            flagInfo={selectedFlag}
-            language={language}
-            setLanguage={setLanguage}
-            apiKey={apiKey}
-            setApiKey={setApiKey}
-            selectedEnvironment={selectedEnvironment}
-            setSelectedEnvironment={setSelectedEnvironment}
-          />
-        )}
-
-        {selectedStep === STEP.VALIDATE_FLAG && selectedFlag && selectedEnvironment && (
-          <ValidateYourFlagView
-            flagInfo={selectedFlag as Feature}
-            language={language as PlatformEntry}
-            apiKey={apiKey as ApiKey}
-            testDone={testDone}
-            setTestDone={setTestDone}
-            environmentIdentifier={selectedEnvironment.identifier}
-          />
-        )}
-      </Container>
-      <Divider />
-      <Layout.Horizontal spacing="small" padding="small">
+        <img src={featureFlagsImg} width="175px" height="50px" />
         <Button
-          text={getString('back')}
-          icon="chevron-left"
+          minimal
+          icon="code-close"
+          width={16}
+          height={16}
+          withoutBoxShadow
           onClick={() => {
-            trackEvent(FeatureActions.GetStartedPrevious, {
-              category: Category.FEATUREFLAG
-            })
-            onPrevious()
+            history.push(routes.toCFOnboarding({ accountId: accountIdentifier, orgIdentifier, projectIdentifier }))
           }}
         />
-        <Button
-          text={getString(selectedStep !== STEP.VALIDATE_FLAG ? 'next' : 'cf.onboarding.complete')}
-          rightIcon={selectedStep !== STEP.VALIDATE_FLAG ? 'chevron-right' : undefined}
-          intent={Intent.PRIMARY}
-          variation={ButtonVariation.PRIMARY}
-          disabled={disableNext}
-          onClick={() => {
-            if (selectedStep === STEP.SELECT_ENV_SDK) {
-              trackEvent(FeatureActions.SetUpYourApplicationVerify, {
-                category: Category.FEATUREFLAG
-              })
-            } else if (selectedStep === STEP.VALIDATE_FLAG) {
-              trackEvent(FeatureActions.TestYourFlagBack, {
-                category: Category.FEATUREFLAG
-              })
-            } else {
-              trackEvent(FeatureActions.GetStartedNext, {
-                category: Category.FEATUREFLAG
-              })
-            }
-            onNext()
-          }}
-        />
-        <FlexExpander />
       </Layout.Horizontal>
-    </Layout.Vertical>
+      <Layout.Horizontal className={css.fullScreenContainer} padding={{ left: 'huge' }}>
+        <Layout.Vertical
+          padding={{ top: 'large', right: 'huge', left: 'huge', bottom: 'none' }}
```

Comment:
You don't need to specify `bottom: 'none'` as that will be the default.
```suggestion
          padding={{ top: 'large', right: 'huge', left: 'huge' }}
```

Reply:
Sweet, removed :)